### DISCUSSION TITLE
从应用接口视角看存储系统

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#215] [Announcements]
https://xuanwo.io/2021/06-storage-system-from-interface/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by reuixiy** [comment]
感谢，之前工作需要部署过 Ceph，看完这篇文章对当时遗留的一些朦胧概念和存储本身都更加清晰了。

**Comment by enkerewpo** [comment]
赞！
